### DISCUSSION TITLE
Wrong cuts

### DISCUSSION THREAD

**Discussion by Risotti** [mifi/lossless-cut#1150] [Support]
A video 7.53 long. Cut down to 3.05. Result after merge & export: 3.57. This has started to happen recently every time rendering the app unusable. It simply doesn't cut of the segments as it should.

**Comment by mifi** [comment]
Hi. I think it's working as intended, because it can only cut on keyframes. See [Known issues & limitations](https://github.com/mifi/lossless-cut#known-issues--limitations)

**Comment by Risotti** [comment]
I have used the program for over a year. I have the latest version now and
have experienced the problem for some weeks. I don't have this problem with
a similar program and I'm on the same pc with the same version of windows.
So, it seems to me that the problem has started with an updated version of
the program.

lør. 21. maj 2022 20.36 skrev Mikael Finstad ***@***.***>:

**Reply by mifi** [reply]
Could you try to install the previous version and confirm that the problem goes away? Also which version number works as expected?

**Comment by Risotti** [comment]
I can't find a previous version on the site?

lør. 21. maj 2022 21.16 skrev Mikael Finstad ***@***.***>:

**Reply by mifi** [reply]
where did you download losslesscut from? all releases can be found here: https://github.com/mifi/lossless-cut/releases

**Comment by Risotti** [comment]
I know. But where are the download links to older versions? I see the
descriptions but no links to download except for the latest release.

lør. 21. maj 2022 22.35 skrev Mikael Finstad ***@***.***>:

**Reply by mifi** [reply]
click "assets"
